Runtime/UIElements/Controls/UIUtils.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+static class UIUtils
+    {
+        static readonly string[] s_SizeUnits = { "KB", "MB", "GB", "TB" };
+
+        public static void SetElementDisplay(VisualElement element, bool value)
+        {
+            if (element == null)
+                return;
+
+            element.style.display = value ? DisplayStyle.Flex : DisplayStyle.None;
+            element.style.visibility = value ? Visibility.Visible : Visibility.Hidden;
+        }
+
+        public static void SetElementDisplayNonEmpty(TextElement element)
```

Comment:
I also not sure why to have another method for this.
Plus method semantics does not really tell me what the method would do.